functests/4_latency/latency.go

```diff
@@ -167,6 +167,78 @@ var _ = Describe("[performance] Latency Test", func() {
 			}
 		})
 	})
+
+	Context("with the cyclictest image", func() {
+		BeforeEach(func() {
+			testName = "cyclictest"
+			initMaximumLatencyValue(testName)
+
+			if profile.Spec.CPU.Isolated == nil {
```

Comment:
should this be made a common check? maybe merged with the check we currently do in `initMaximumLatencyValue`?

Reply:
Not for all tools. 
For example, ```hwlatdetect``` doesn't care about this check.

Reply:
ack